Genome-Annotation/tutorials/general-introduction.md

```diff
@@ -1,110 +1,154 @@
-GALAXY WORKSHOP on GENOME ANNOTATION
-------------------------------------
+---
+layout: tutorial_hands_on
+topic_name: Genome-Annotation
+tutorial_name: general-introduction
+---
  
+# Introduction
+
 Genome annotation is the process of attaching biological information to sequences.
 It consists of three main steps:
 
  - identifying portions of the genome that do not code for proteins
  - identifying elements on the genome, a process called gene prediction, and
  - attaching biological information to these elements.
 
-
-# 1. Introduction into File Formats
+> ### Agenda
+>
+> In this tutorial, we will deal with:
+>
+> 1. [Introduction into File Formats](#introduction-into-file-formats)
+> 2. [Structural Annotation](#structural-annotation)
+> 3. [Functional Annotation](#functional-annotation)
+{: .agenda}
+ 
+# Introduction into File Formats
 
 **FASTA**
 
-DNA and protein sequences are written in FASTA format where you have in the first line a ">" followed by the description. In the second line the sequence starts.
-
+> DNA and protein sequences are written in FASTA format where you have in the first line a ">" followed by the description. In the second line the sequence starts.
 
 ![FASTA file](../images/fasta_format.png)
 
-
-
 **GFF3**
 
-The general feature format (gene-finding format, generic feature format, GFF) is a file format used for describing genes and other features of DNA, RNA and protein sequences.
+> The general feature format (gene-finding format, generic feature format, GFF) is a file format used for describing genes and other features of DNA, RNA and protein sequences.
 
 <img src="../images/gff3_format.png" width="70%">
 
-
-
 **GENBANK**
 
-The genbank sequence format is a rich format for storing sequences and associated annotations.
+>The genbank sequence format is a rich format for storing sequences and associated annotations.
 
 ![genbank file](../images/gb_full.png)
 
+# Structural Annotation
 
-# 2. Structural Annotation
-
-For the genome annotation we us a piece of the *Aspergillus fumigatus* [genome sequence](../input_data/Aspergillus_sequence.fasta) as input file. 
-
-
-
-## 2.1 Sequence Features
+> ### :nut_and_bolt: Comments
+> For the genome annotation we use a piece of the *Aspergillus fumigatus* [genome sequence](../input_data/Aspergillus_sequence.fasta) as input file. 
+{: .comment}
 
-First we want to get some general information about our sequence. Count the number of bases in your sequence (**compute sequence length**), and check for sequence composition and GC content (**geecee**). Plot the sequence composition as bar chart. 
+## Sequence Features
 
-<img src="../images/barchart_sequencecomposition.png" width="30%">
+> ### :pencil2: Hands-on: Sequence composition
+>
+> First we want to get some general information about our sequence.
```

Comment:
Can you move this sentence outside of the box? It is more an introduction to the hands-on